packages/documentation/docs/controls/_content-header_styleguide.md

```diff
@@ -0,0 +1,46 @@
+The content header provides a brief overview of the content on a page. It helps the user to understand what the page is about.
+We typically use it at the very top of the page.
+
+![Content header overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2250-4784&mode=design&t=XmCepM9jPR9PImPw-4)
+ 
+1. Back button 
+2. Content headline
+3. Content subtitle
+4. Action buttons
+
+## Variants
+With our content header variants, you can make it easer for the user to follow the hierarchy of the page.
```

Comment:
Maybe something like "...easier for you to achieve a well-balanced visual hierarchy throughout the page" - users don't follow a hierarchy :)